bazelrunner/src/main/java/org/jetbrains/bsp/bazel/bazelrunner/BazelProcess.kt

```diff
@@ -13,9 +15,11 @@ class BazelProcess internal constructor(
     private val originId: String?
 ) {
 
-    fun waitAndGetResult(): BazelProcessResult {
-        val outputProcessor = AsyncOutputProcessor(process, logger.withOriginId(originId)::message, LOGGER::info)
+    fun waitAndGetResult(ensureAllOutputRead: Boolean = false): BazelProcessResult {
```

Comment:
I would prefer to default to `true` 

cc @abrams27

Reply:
can be true for me, but then we need to set it to false in the `ExecuteService`

Reply:
true is just safer

Reply:
imo we probably shouldn't be trying to parse the output at all for anything, and use BEP/BES instead.  We could get the execution and workspace roots from the BEP stream as well, and then there'd be no need to have a sync version of this at all.